src/qt3utils/math_utils.py

```diff
@@ -0,0 +1,9 @@
+from scipy.ndimage import uniform_filter1d
+
+
+def get_rolling_mean(data, window_size=5):
+    """
+    Rolling mean is a good way to look at the average behavior of noisy data.
+    It represents the local average value within a given window.
+    """
+    return uniform_filter1d(data, size=window_size)
```

Comment:
why not just call scipy.uniform_filter1d directly? 

Also, this is only used once during instantiation. Do you think we need this as a separate module?

Reply:
Honestly, I started the math_utils file with the intention to add more methods there in the future. I saw a lot of math-related methods in different files, and thought it might be nice to round them all up. What do you think?